lib/docs-builder.js

```diff
@@ -97,7 +96,7 @@ function build(srcPatterns, destDir, options, callback) {
 	}
 
 	function writeMeta() {
-		const meta = Object.values(b.get());
+		const meta = _.values(b.get());
```

Comment:
Why lodash here?

Reply:
`_.values` is used in other places anyways and `Object.values` is only supported since node 7. This simple change should make the library node 6 compatible.

Reply:
Ok